app/javascript/crayons/Modal/__stories__/Modal.stories.jsx

```diff
@@ -46,6 +59,13 @@ export default {
         defaultValue: { summary: false },
       },
     },
+    sheet: {
+      description:
+        'Special style to display the modal as full view height. Useful for larger chunks of content, and may be anchored to left or right of screen usingn the `align` prop',
+      table: {
+        defaultValue: { summary: false },
+      },
+    },
     centered: {
       description:
         'In some cases it might be "nicer" to center Modals content. This will only work with `prompt` though.',
```

Comment:
Quick, non-blocking suggestion: let's make this singular and lowercase it to keep it consistent with the singular "modal" above on line `64`:
```suggestion
        'In some cases it might be "nicer" to center modal content. This will only work with `prompt` though.',
```